core/src/main/resources/application.yml

```diff
@@ -180,6 +180,8 @@ cb:
                 CDP 1.2 - Operational Database: Apache HBase=cdp-opdb-702;
                 CDP 1.2 - SDX Light Duty: Apache Hive Metastore, Apache Ranger, Apache Atlas=cdp-sdx-702;
                 CDP 1.2 - SDX Medium Duty: Apache Hive Metastore, Apache Ranger, Apache Atlas=cdp-sdx-medium-ha-702;
+                CDP 1.3 - Streams Messaging Cluster=cdp-streaming-703;
+                CDP 1.3 - Streams Messaging Cluster (small)=cdp-streaming-small-703;
```

Comment:
what small means? Is this the final name ?

Reply:
Small means:
- There are no dedicated hosts for ZK
- Cheaper instances with less CPU and also less RAM for brokers.

Intension is to have a low-cost option for piloting / developing with Kafka.

Name was coined by @gergowilder and me, product management may want to change it.

Reply:
Synched with Matt Schumpert on naming. Here's what we wanna do:

- Cluster Definition: "CDP 1.2 - Streams Messaging Light Duty"
- Cluster Template: "CDP 1.2 - Streams Messaging Light Duty: Apache Kafka, Schema Registry, Streams Messaging Manager"

- Cluster Definition: "CDP 1.2 - Streams Messaging Heavy Duty"
- Cluster Template: "CDP 1.2 - Streams Messaging Heavy Duty: Apache Kafka, Schema Registry, Streams Messaging Manager"

The CDP version needs to be adjusted to the right Runtime version in production when we release our cluster definition

Reply:
Hey @mikchaos, one more question re the display name that appears in CM. I checked the other blueprints (Data Mart, LLAP, ...) and this name is kind of simple (no spaces, only lowercase letters), such as: **"de-llap**, **datamart**. Ours used to be **CDP 1.x Streams Messaging Cluster** but I changed it to **streams-messaging** so that it'll be similar to the others. Let me know your thoughts.

Reply:
sounds good to me